src/jit/simd.cpp

```diff
@@ -2879,6 +2882,18 @@ GenTree* Compiler::impSIMDIntrinsic(OPCODE                opcode,
             op2 = impSIMDPopStack(simdType);
             op1 = impSIMDPopStack(simdType, instMethod);
 
+#ifdef _TARGET_XARCH_
+            if (simdIntrinsicID == SIMDIntrinsicBitwiseAndNot)
```

Comment:
This is the only new change and it ensures that the operands are passed in the expected order for XARCH, since it does `~op1 & op2`, while ARM and the software fallback do `op1 & ~op2`.